Fix/1395 gallery spacing

pardhan03 opened the pull request:
Fixes #1395 : Extra Spacing in Gallery When Images and Files Are Attached

This PR addresses a UI issue where extra spacing was introduced around the gallery component when both images and files (e.g., image or documents) were attached in a single message.

🧩 Fix

  - Adjusted layout logic and spacing in FileAttachment.tsx.
  - Ensured that mixed media (files + images) render in a consistent and visually correct layout.
 
### After Fix

![solution](https://github.com/user-attachments/assets/1e2433a1-83b9-48c0-9a21-265c2faf6729)

![solution1](https://github.com/user-attachments/assets/684f7344-ac6e-486b-b91b-08e586d20f33)

pardhan03 committed:
Merge pull request #1 from GetStream/develop

Getting the latest changes

pardhan03 committed:
fix: resolve extra spacing in gallery when images and files are attached together

khushal87 commented:
<img width="1206" height="2622" alt="simulator_screenshot_765058C4-2104-426B-AD6C-BD52EC58D506" src="https://github.com/user-attachments/assets/0c9738cc-eeeb-421a-b2de-a75f5d2d8825" />

Hey the fix Doesn't seem to be correct as it trims the file attchment width to a very small one. I will look into it and try to add a new fix. Probably we need to center the image instead of playing with the file attachment height.

pardhan03 commented:
@khushal87 I removed the maxWidth property and added flex: 1, so now the component takes up the full width of its parent. To control the minimum size, we can set a minWidth property. This way, it won’t shrink too small if the parent has limited space. However, when using minWidth, the image may end up with extra space around it depending on how it scales.

isekovanic commented:
Hi @pardhan03 ,

Really sorry for the terribly long wait here. Since we had released V9 this fix should be out of scope. The new gallery is done in correspondence with our general design and is what is used across all SDKs now, so we won't explicitly be changing this behaviour directly in the SDK as it should be stable. If you'd like something specific, please feel free to use our theming system to achieve it (or let us know if it's not currently possible to do so so that I can extend the theme) !

Thanks a lot for your contribution anyway, always greatly appreciated.

Take care !
